pkg/prometheus/clickhouseprometheus/client.go

```diff
@@ -3,6 +3,8 @@ package clickhouseprometheus
 import (
 	"context"
 	"fmt"
+	value2 "github.com/prometheus/prometheus/model/value"
```

Comment:
please use some sensible name - "value2" is very random and move the import to the rest of the prom imports below.